views.py

```diff
@@ -13,14 +13,13 @@ async def read_number_of_planets():
         return {"Number of Planets": planets["count"]}
 
 
-@app.post("/movie")
+@app.post("/movie/{movie_id}")
 async def get_movie_data(movie_id: int):
     response = requests.get(f"https://swapi.py4e.com/api/films/{movie_id}/")
-    if response.status_code != 200:
-        return {"error": f"Movie with ID {movie_id} not found."}
     movie_data = response.json()
     people = [
```

Comment:
This line will wait synchronously for all requests to finish. Please use asynchronous lib httpx with chain https://realpython.com/async-io-python/#chaining-coroutines